templates/sectionEditor/viewDraftDecision.tpl

```diff
@@ -0,0 +1,70 @@
+{**
+ * templates/sectionEditor/submissionReview.tpl
+ *
+ * Copyright (c) 2013 Simon Fraser University Library
+ * Copyright (c) 2003-2013 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Submission review.
+ *
+ *}
+{strip}
+{translate|assign:"pageTitleTranslated" key="submission.page.review" id=$submission->getId()}{assign var="pageCrumbTitle" value="submission.review"}
+{include file="common/header.tpl"}
+{/strip}
+
+<ul class="menu">
+	<li><a href="{url op="submission" path=$submission->getId()}">{translate key="submission.summary"}</a></li>
+	<li class="current"><a href="{url op="submissionReview" path=$submission->getId()}">{translate key="submission.review"}</a></li>
+	{if $canEdit}<li><a href="{url op="submissionEditing" path=$submission->getId()}">{translate key="submission.editing"}</a></li>{/if}
+	<li><a href="{url op="submissionHistory" path=$submission->getId()}">{translate key="submission.history"}</a></li>
+	<li><a href="{url op="submissionCitations" path=$submission->getId()}">{translate key="submission.citations"}</a></li>
+</ul>
+
+<h3>{translate key="editor.draft.decision"}</h3>
+<div>
+	<form method="POST">
+		{if $view == 'view'}
+			<p>{translate key="editor.draft.intro"}</p>
+			{foreach from=$draft item=d}
+				{assign var="decision" value=$d.decision}
+				<h5>{translate key="user.role.editor"}:</h5>
+				{$d.junior_editor_name}
+				<h5>{translate key="editor.article.recommendation"}:</h5>
+				{translate key=$editorDecisionOptions.$decision}
+				<h5>{translate key="editor.draft.SectionEditorRecommendation"}:</h5>
+				{$d.note}
```

Comment:
This needs to be escaped to prevent XSS problems. (Also elsewhere in this template)